draft-ietf-quic-transport.md

```diff
@@ -2206,19 +2206,17 @@ An endpoint MUST NOT reduce their MTU below this number, even if it receives
 signals that indicate a smaller limit might exist.
 
 Clients MUST ensure that the first packet in a connection, and any
-retransmissions of those octets, has a QUIC packet size of least 1232 octets for
-an IPv6 packet and 1252 octets for an IPv4 packet.  In the absence of extensions
-to the IP header, padding to exactly these values will result in an IP packet
-that is 1280 octets.
+retransmissions of those octets, has a QUIC packet size of least 1200 octets.
 
-The initial client packet SHOULD be padded to exactly these values unless the
-client has a reasonable assurance that the PMTU is larger.  Sending a packet of
-this size ensures that the network path supports an MTU of this size and helps
-reduce the amplitude of amplification attacks caused by server responses toward
-an unverified client address.
+The initial client packet SHOULD be padded to exactly 1200 octets unless the
+client has a reasonable assurance that the PMTU is larger.  This size includes
+the QUIC header and integrity check, but not the UDP or IP header. Sending a
+packet of this size ensures that the network path supports an MTU of this size
+and helps reduce the amplitude of amplification attacks caused by server
+responses toward an unverified client address.
 
 Servers MUST ignore an initial plaintext packet from a client if its total size
```

Comment:
It'd be nice to come up with a consistent way of referring to this "total size" which does not include IP+UDP.